### ISSUE TITLE
Suggestion reguarding droping slider ends in laser

### ISSUE THREAD

**Issue by RadekMojzis** [ppy/osu#11634]
I am  not sure where  to post stuff like this, there are a couple dead posts on the osu forum.
Ive recently seen this [interview with Peppy ](https://www.youtube.com/watch?v=Tsjx5SkHsfE&ab_channel=KoiFishu) he mentioned that he wants to push through with the slider ends breaking combo.

On one hand i agree that droping slider ends should be more punishing. On the other hand, making a change so fundamental would basically break like half of all FCs. 

My suggestion would be to intead of penalising the player by breaking the combo, to reward perfect combos by a PP/score boost, that way you dont break old scores and you can seamlessly blend score between clients. With that said, i want to let you know that im switching to laser permanently :)

TLDR giff more pp for perfect combo instead of combo breaking on missed slider end.

**Comment by peppy** [comment]
i'm not sure what this is related to. slider ends are not considered for judgement in the current master.
